src/shared/components/Footer/FooterList/component.tsx

```diff
@@ -1,16 +1,9 @@
 import React from 'react';
-import { FooterListWrapper, FooterListHeading, SocialIconWrapper, FooterListItem, SocialIconLink } from './styles';
-import { TwitterIcon, LinkedinIcon, FacebookIcon } from 'shared/icons';
 
-type FooterItem = {
-  name: string;
-  link: string;
-};
+import { MenuItem, MenuList } from 'shared/types';
 
-type FooterList = {
-  heading: string;
-  list: FooterItem[];
-};
+import { FooterListWrapper, FooterListHeading, SocialIconWrapper, FooterListItem, SocialIconLink } from './styles';
+import { TwitterIcon, LinkedinIcon, FacebookIcon } from 'shared/icons';
```

Comment:
I think this import could be a bit higher :)

Reply:
I did not understand you a little. It looks like this for me
<img width="910" alt="Screenshot 2020-04-22 at 11 58 02" src="https://user-images.githubusercontent.com/27691774/79962309-8d5c1280-8490-11ea-9934-8898fff5e8bd.png">

Reply:
Due to our code style, absolute imports should go higher than relative

Reply:
Donee!